sound_cloud_backend/sound_cloud_backend/restconf/main.py

```diff
@@ -1,6 +1,6 @@
 REST_FRAMEWORK = {
     'DEFAULT_AUTHENTICATION_CLASSES': [
-        'rest_framework_simplejwt.authentication',
```

Comment:
Any specific reason why you moved from jwt authentication to django token authentication?

Reply:
I can't remember why i replaced jwt with TokenAuthentication.